test/tsd.ts

```diff
@@ -106,6 +106,25 @@ function testDataQueryBuilderClass() {
     const query: Backendless.DataQueryValueI = dataQuery.build();
 }
 
+function testDataJSONUpdateBuilder() {
+    const jsonUpdateBuilder = Backendless.JSONUpdateBuilder
+
+    const argKey: string = '$.number'
+    const argValue: number = 11
+
+    let jsonPayload: object = jsonUpdateBuilder.SET().addArgument(argKey, argValue).toJSON()
```

Comment:
add `jsonUpdateBuilder = jsonUpdateBuilder.SET().addArgument(argKey, argValue)` as well for each operation

Reply:
+